[mainnet] Optimism's review on transfer & delegate OP tokens task (#129)
base-org/contract-deployments: mainnet/2024-02-23-transfer-op/README.md

Now click on the "State" tab. Verify that:

1. Verify that the nonce is incremented for the Nested Multisig under the "GnosisSafeProxy" at address `0x0a7361e734cf3f0394b0fc4a45c74e7a4ec70940`:

```
Key: 0x0000000000000000000000000000000000000000000000000000000000000005
Before: 0x0000000000000000000000000000000000000000000000000000000000000001
After: 0x0000000000000000000000000000000000000000000000000000000000000002
```

> 2. And for the same contract, verify that this specific execution is approved by the Coinbase Multisig: 

Author: ```suggestion
2. And for the same contract, verify that this specific execution is approved by the other signer multisig: 
```

```

> Key: 0x11df082a13e3abe7c86905df73f40491ddca07ffdd5dd589ca60b5cd151a98f5

Author: ```suggestion
Key (if you are an OP signer): 0x11df082a13e3abe7c86905df73f40491ddca07ffdd5dd589ca60b5cd151a98f5
Key (if you are a CB signer): 0xc5af7fb02845bd01e3c5d7620938079cb2f0eb79259fc52740bbe4820401d909
```

Before: 0x0000000000000000000000000000000000000000000000000000000000000000
After: 0x0000000000000000000000000000000000000000000000000000000000000001
```

3. Verify that the nonce is incremented for the OP Foundation Multisig under the "GnosisSafeProxy" at address `0x2501c477D0A35545a387Aa4A3EEe4292A9a8B3F0`:

```
Key: 0x0000000000000000000000000000000000000000000000000000000000000005
Before: 0x0000000000000000000000000000000000000000000000000000000000000077
After: 0x0000000000000000000000000000000000000000000000000000000000000078

> ```

Author: suggestion: 
3. Verify that the nonce is incremented for your multisig.

 For the OP Foundation Multisig under the "GnosisSafeProxy" at address `0x2501c477D0A35545a387Aa4A3EEe4292A9a8B3F0`:
```
Key: 0x0000000000000000000000000000000000000000000000000000000000000005
Before: 0x0000000000000000000000000000000000000000000000000000000000000077
After: 0x0000000000000000000000000000000000000000000000000000000000000078
```
 Or for the Coinbsae Multisig under the address `0x6e1dfd5c1e22a4677663a81d24c6ba03561ef0f6`:
```
Key: 0x0000000000000000000000000000000000000000000000000000000000000005
Before: 0x0000000000000000000000000000000000000000000000000000000000000001
After: 0x0000000000000000000000000000000000000000000000000000000000000002
```

  }
}
```

7. Verify that the `L1FeeVault` at `0x420000000000000000000000000000000000001a` receives the gas associated with the call:

> _Note: the balance change here might be different depending on network traffic_

Author: ```suggestion
_Note: the balance values here will not be exactly the same_
```
